src/lib/commands/remoteFs.ts

```diff
@@ -75,9 +82,142 @@ export const registerCommands = (context: ExtensionContext) => {
         if (!selection || selection === 'Cancel') {
           return;
         }
-        const res = await deleteFile(getPortalId(), filePath);
-        commands.executeCommand('hubspot.remoteFs.refresh');
+        deleteFile(getPortalId(), filePath).then(() => {
+          window.showInformationMessage(`Successfully deleted ${filePath}`);
+          invalidateParentDirectoryCache(filePath);
+          commands.executeCommand('hubspot.remoteFs.refresh');
+        });
+      }
+    )
+  );
+  context.subscriptions.push(
```

Comment:
I think I'm following the logic branching here - however the cyclomatic complexity is a bit high. Would it be easier to understand/maintain if we could break out some of these branches in their own functions? That would also allow the function name to help describe what the particular logic branch aims to do.